media/js/firefox/australis/common.js

```diff
@@ -68,10 +43,26 @@
 
     // link directly to Firefox Accounts when clicking the Sync CTA button
     Mozilla.UITour.getConfiguration('sync', function (config) {
-        $('.sync-cta').on('click', '.button', trackSyncClick);
+        $('.sync-cta .button').each(function() {
+            $(this).attr({
+                'data-page-name': pageId,
+                'data-interaction': 'button click',
+                'data-element-location': 'Get Started with Sync',
+            }).on('click', Mozilla.UITour.showFirefoxAccounts);
+        });
     });
 
-    // track learn more links on click
-    $('.learn-more a').on('click', trackLearnMoreLinks);
+    // track learn more links
+    $('.learn-more a').each(function() {
+        $(this).attr({
+            'data-page-name': pageId,
+            'data-interaction': 'button click',
+            'data-element-location': 'Get Started with Sync',
+        }).on('click', function() {
```

Comment:
Should explicitly pass the `e` argument to this function (since it's used in the function body).